File: Python/password_validation.py
p = input()

sc=['!', '@', '#', '$', '%', '&', '*']
num=['1','2','3','4','5','6','7','8','9','0']
c=0
n=0
ls = []
ls= ls + list(p)

for i in range(len(p)):
    if ls[i] in sc:
        c=c+1
    if ls[i] in num:
        n=n+1

if len(p)<7:
    print("Weak")
elif len(p)>=7 and n>1 and c>1:
    print("Strong")
else:
    print("Weak")

File: Golang/password_validation.go
package main

import (
       "fmt"
       "unicode"
       "strings"
       )

func main() {
  var pw string
  fmt.Scanln(&pw)
  str1 := "!@#$%&*"
  a := 0
  b := 0
  c := 0
  for _, l := range pw { 
     a += 1 
          
     if unicode.IsDigit(l) {
         b += 1
     }
     if strings.Contains(str1, string(l))  {
         c += 1
       }
  }            
  if (b > 1 && c > 1 && a > 6)   {    
     fmt.Print("Strong")
     } else {
     fmt.Print("Weak")
    }
}
